### ISSUE TITLE
Support for doctrine/mongodb-odm: ^2.0

### ISSUE THREAD

**Issue by rbaarsma** [doesntmattr/mongodb-migrations-bundle#27]
I'm trying to install this bundle on Symfony 4 with doctrine/mongodb-odm-bundle 4.0.0 and doctrine/mongodb-odm 2.0.0, but unfortunately this is not possible due to the composer requirement in this bundle for `doctrine/mongodb-odm: ^1.0`.

It would be great if this can be upgraded to `doctrine/mongodb-odm: ^1.0|^2.0`, hoping the bundle itself is also compatible with doctrine/mongodb-odm 2.0

**Comment by beloglazov91** [comment]
@rbaarsma You were right: it helps!

I did a fork and tested it. You can test my solution right now:

1. add to your composer.json the following
```
"repositories": [
        {
            "type": "git",
            "url": "https://github.com/beloglazov91/mongodb-migrations-bundle"
        }
]
```

2. Install the dependency from the repository above:
```
composer require "doesntmattr/mongodb-migrations-bundle=dev-master"
```

**Comment by caciobanu** [comment]
Support has landed in 3.0.
